src/boot/efi/stub.c

```diff
@@ -253,29 +254,113 @@ static EFI_STATUS load_addons_from_dir(
 
 }
 
-static EFI_STATUS cmdline_append_and_measure_addons(
+static void cmdline_append_and_measure_addons(
+                char16_t *cmdline,
+                char16_t **cmdline_append,
+                bool *ret_parameters_measured) {
+
+        _cleanup_free_ char16_t *tmp = NULL;
+        bool m = false;
+
+        assert(ret_parameters_measured);
+        assert(cmdline_append);
```

Comment:
doesn't matter, but we usually add the asserts for func params in the order the function takes the parameters. i.e. swap these two lines?